include/WindowSlider/SplitJoinWS.h

```diff
@@ -0,0 +1,80 @@
+/*! \file SplitJoinWS.h*/
+//
+// Created by tony on 18/03/22.
+//
+
+#ifndef _WINDOWSLIDER_SPLITWS_H_
+#define _WINDOWSLIDER_SPLITWS_H_
+
+#include <WindowSlider/AbstractWS.h>
+#include <Utils/AbstractC20Thread.h>
+#include <JoinProcessor/SplitJoinJP.h>
+using namespace INTELLI;
+using namespace std;
+namespace INTELLI {
+/**
+ * @ingroup WINDOWSLIDER_EAGER
+* @class SplitJoinWS WindowSlider/SplitJoinWS.h
+* @brief The eager window slider of split join
+* @note
+* detailed description:
+To init and run, follow the functions below to start a WS
+  \li Configure the window type, time or count, @ref setTimeBased
+  \li Configure window length: @ref setWindowLen
+  \li Configure slide length: @ref setSlideLen (default is 1 if not called)
+  \li Set parallel executing behavior on SMP,@ref setParallelSMP
+  \li Optional, (@ref setRunTimeScheduling)
+  \li To make the parallel join processors started, @ref initJoinProcessors
+  \li Feed tuples @ref feedTupleS or @ref feedTupleR
+  \li Terminate, by @ref terminateJoinProcessors
+*
+*/
+class SplitJoinWS : public AbstractWS, public AbstractC20Thread {
```

Comment:
The name is werid. AbstractC20Thread?

Reply:
A basic thread class under C20 style, see Utils